e2e/systemsched/input/system_job0.nomad

```diff
@@ -0,0 +1,53 @@
+job "redis" {
+  datacenters = ["dc1"]
+
+  type = "system"
+
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
+
+  group "cache" {
+    count = 1
+
+    restart {
+      attempts = 10
+      interval = "1m"
+
+      delay = "2s"
+      mode  = "delay"
+    }
+
+    task "system_task" {
+      driver = "docker"
+
+      config {
+        image = "bash:latest"
+
+        command = "bash"
+        args    = ["-c", "sleep 15000"]
+      }
+
+      env {
+        version = "1"
+      }
+
+      logs {
```

Comment:
There are a few little things in the test input files that can make the purpose of the test less clear. If you find these as you're reading, you end up having to dig through the test to see if they're intentional or coincidental:
- Test inputs include unused stanzas like `logs`. 
- The memory/cpu comments don't match the values.
- The name of the job is `redis` but it's not Redis. (Same for the `taskgroup.name`)

Reply:
thanks :( definitely should be more clear, updating